### DISCUSSION TITLE
Some apps start very slowly after switching to Regolith on Archlinux

### DISCUSSION THREAD

**Discussion by aleb** [regolith-linux/regolith-desktop#948] [Q&A] [answered]
For example the EoG picture viewer, or Firefox. Before they took almost no time to start.

Did anybody deal with this issue?

**Comment by aleb** [comment]
I found some workarounds at https://stackoverflow.com/a/77490432/804479

**Comment by kgilmer** [comment, marked as answer]
Hi @aleb , yes this is an active issue.  My plan is to implement a portals backend for Regolith.  I believe this is required to allow Regolith to function if the `xdg-desktop-portal-gnome` package is also installed.  I'm not aware of a workaround in which both that package and the Regolith DE may function with GNOME 45.

https://github.com/regolith-linux/regolith-desktop/issues/936
